inc/utils/content.php

```diff
@@ -564,17 +581,19 @@ function get_categories_array($post){
 
  function odm_echo_extras($postID = "") {
  	 $postID = $postID ? $postID : get_the_ID();
-	 if (function_exists('get_post_meta')) :
-		 $get_author = get_post_meta($postID, 'author', true);
-		 $get_localized_author = get_post_meta($postID, 'author_'.odm_language_manager()->get_current_language(), true);
-	   if ($get_author != '' || $get_localized_author != ''):
-	     $news_source_info = '<span class="lsf">&#xE041;</span> ';
-	     if ($get_localized_author != ''):
-	         $news_source_info .= $get_localized_author.'<br />';
-	     else:
-	         $news_source_info .= $get_author.'<br />';
-	     endif;
-	   endif;
+	 if (function_exists('get_post_meta')):
```

Comment:
@acorbi Thank you for the explanation. Currently public announcement sources aren't exposed in the interface for ODM, though we do have one for news. I'll open a feature request for this.

@ChandaraKhun @Huyeng What do you think of my suggestion to display the name of the source rather than the type of the source for announcements on the home page (see screenshot above)? To me the current implementation is a little inconsistent and could inform people better with the full name of the source rather than type.